### PR TITLE
Remove unused SCSS rules from mofo-bootstrap

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #4546

Remove SCSS imports that I'm certain fo.mo doesn't utilize. More cleanup to come in follow-up PRs.

Wooohooo and compiled CSS is now down by ~170KB.

### CODE REVIEW

File: source/sass/global.scss [hunk 1]
```diff
@@ -60,15 +60,6 @@ button:hover {
   }
 }
 
-.mofo-footer {
```
Comment (mmmavis) [inline source/sass/global.scss:63]: No DOM element in the codebase has this classname.

File: source/sass/mofo-bootstrap/mofo-bootstrap.scss [hunk 2]
```diff
@@ -26,39 +26,22 @@
...
-@import "../../../node_modules/bootstrap/scss/input-group";
 @import "../../../node_modules/bootstrap/scss/custom-forms";
 @import "../../../node_modules/bootstrap/scss/nav";
 @import "../../../node_modules/bootstrap/scss/navbar";
+// TODO: utilizing Bootstrap's 'card' rules instead of writing our own custom SCSS for cards on fo.mo
 @import "../../../node_modules/bootstrap/scss/card";
-@import "../../../node_modules/bootstrap/scss/breadcrumb";
-@import "../../../node_modules/bootstrap/scss/pagination";
-@import "../../../node_modules/bootstrap/scss/badge";
-@import "../../../node_modules/bootstrap/scss/jumbotron";
```
Comment (Pomax) [inline source/sass/mofo-bootstrap/mofo-bootstrap.scss:45]: 🐘 ?

File: source/sass/mofo-bootstrap/overrides/_mixins.scss [hunk 3]
```diff
@@ -28,8 +28,7 @@
   }
 
   &:active,
-  &.active,
-  .open > &.dropdown-toggle {
+  &.active {
```
Comment (Pomax) [inline source/sass/mofo-bootstrap/overrides/_mixins.scss:31]: There's a whole bunch of "double rules" in this file now.

Comment (mmmavis) [reply source/sass/mofo-bootstrap/overrides/_mixins.scss:31]: Ah. One is targeting for pseudo class `:active` and one is targeting `.active` class.

There are multiple rulesets for them in this file but each is wrapped in a different mixin.

Comment (Pomax) [reply source/sass/mofo-bootstrap/overrides/_mixins.scss:31]: ahhhh

**Pull request merged**